src/assets/constants/colors.ts

```diff
@@ -5,32 +5,52 @@ export const reviewColors = {
 };
 
 export const coursesColorSet = [
-  {
-    text: 'Gray',
```

Comment:
Do we want users to still have gray courses in their plan, even though we do not offer the color anymore?

Reply:
Maybe we can comment out gray for now, because I think we might want to reserve it for placeholder for templates?

Reply:
Even if we do not offer gray color anymore, subject colors for each user are saved in firestore so existing users would be able to use gray color unless they make a change by themselves.